examples/src/main/java/com/flipkart/gjex/examples/helloworld/web/HelloWorldResourceConfig.java

```diff
@@ -31,10 +32,14 @@
 public class HelloWorldResourceConfig extends ResourceConfig {
 	
 	@Inject
-	public HelloWorldResourceConfig (HelloWorldResource1 helloWorldresource1, 
-			HelloWorldResource2 helloWorldresource2) {
+	public HelloWorldResourceConfig (HelloWorldResource1 helloWorldresource1,
+																	 HelloWorldResource2 helloWorldresource2,
+																	 RotationManagementResource rotationManagementResource,
```

Comment:
`RotationManagementResource` should be by default added, and not required to be explicitly added. Lets change the behavior.

Reply:
added by default